.github/workflows/test-action.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
         with:
           path: |
             ${{ github.workspace }}/node_modules
-          key: ${{ runner.os }}-node-16-${{ hashFiles('package-lock.json') }}
+          key: ${{ runner.os }}-node-16-action-${{ hashFiles('yarn.lock') }}
```

Comment:
Does this scoping look right to you? Most of the data is in the root node_modules, but the action and cli builds would result in different things being installed?

Reply:
Yes, looks correct.